tests/cpp/operator/mkldnn.cc

```diff
@@ -571,9 +613,8 @@ void VerifyActResult(const std::vector<NDArray *> &in_arrs, const NDArray &arr)
   mshadow::default_real_t *d1 = static_cast<mshadow::default_real_t*>(blob1.dptr_);
   mshadow::default_real_t *d2 = static_cast<mshadow::default_real_t*>(blob2.dptr_);
   EXPECT_EQ(tmp1.shape().Size(), tmp2.shape().Size());
-  for (size_t i = 0; i < tmp1.shape().Size(); i++) {
-    EXPECT_EQ(d1[i], std::fmax(d2[i], 0));
-  }
+  for (size_t i = 0; i < tmp1.shape().Size(); i++)
+    ASSERT_EQ(d2[i], std::fmax(d1[i], 0));
```

Comment:
shouldn't we use google test check?

Reply:
ASSERT_EQ is from gtest? I am using ASSERT_EQ over EXPECT_EQ here cause I want the test to fail as soon as one comparison fails (else we get an error message from every incorrect element in the vector).

Reply:
ASSERT_EQ is defined by mxnet. We should use EXPECT_EQ in the unit test because we want to see all failures.